components/module-components/Contact/circle.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import style from './contact.module.scss';
+
+const Circle: React.FC = () => {
+  return (
```

Comment:
we need better naming for this component.
maybe SpinningStiker.tsx or SpinningLogoCircle.tsx